### ISSUE TITLE
do text drags not work in reddit articles?

### ISSUE THREAD

**Issue by tockschmock** [zimocode/smartup#220]
tested in a couple of reddit articles

my Simple Drags / Text Actions:
- Search Texts in New Background Tab
- Copy Texts 
do not work here ???

**Comment by Limboso** [comment]
Yeah, just tested here and same thing for me. Simple drag (text actions) do not work in reddit comments. Outside comments it does work though.
